docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -185,6 +189,12 @@ below shows all its proper subtransactions on the right (yellow boxes).
    :align: center
    :width: 100%
 
+To illustrate contract keys, suppose that the contract key for a `PaintOffer` consists of the reference number and the painter as the key maintainer.
+So Alice can refer to the `PaintOffer` by its key `(P, P123)`.
+To make this explicit, we use the notation `PaintOffer @P A &P123` for contracts, where `@` marks the maintainers and `&` the other key parts.
```

Comment:
The `&` notation doesn't seem to be actually used anywhere in the document; I'd drop it.

Reply:
Have you looked at at the figures? They're full of `&`.

Reply:
As discussed, this is mainly for visualisation in the diagrams, just like $ for signatories and underlines for observers.